lib/make.js

```diff
@@ -15,11 +15,37 @@ function provideBuilder() {
     },
 
     settings: function (cwd) {
-      return [ {
+
+      var makefile = fs.realpathSync(path.join(cwd, 'Makefile'));
+
+      var createBuildConfig = function(name) {
+        return {
+          name: 'GNU Make: ' + name,
+          exec: 'make',
+          args: [ name ],
+          sh: false
+        };
+      };
```

Comment:
Making a function for this and using it from one place just makes this more difficult to comprehend. Either refactor the `config.push` to make use of this function, or use `config.push` in the `forEach`.